config/rhoai/prometheus_rule.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: prometheus-rules
+  namespace: openshift-monitoring
+spec:
+  groups:
+    - name: kueue-critical-alerts
+      rules:
+      - alert: KueuePodDown
+        expr: kube_pod_status_ready{pod=~"kueue-.*", condition="true"} == 0
+        for: 5m
+        labels:
+          severity: critical
+        annotations:
+          summary: "Kueue pod is down ({{ $labels.pod }})"
+          description: "The Kueue pod {{ $labels.pod }} in namespace {{ $labels.namespace }} is not ready."
+    - name: kueue-info-alerts
+      rules:
+      - alert: LowClusterQueueResourceUsage
+        expr: sum(kueue_cluster_queue_resource_usage) by (cluster_queue, resource) / sum(kueue_cluster_queue_nominal_quota) by (cluster_queue, resource) < 0.2
+        for: 1d
+        labels:
+          severity: info
+        annotations:
+          summary: Low {{ $labels.resource }} resource usage in cluster queue {{ $labels.cluster_queue }}
+          description: The {{ $labels.resource }} resource usage in cluster queue {{ $labels.cluster_queue }} is below 20% of its nominal quota for more than 1 day.
+      - alert: ResourceReservationExceedsQuota
+        expr: (sum(kueue_cluster_queue_resource_reservation) by (resource, cluster_queue)) / 10 > (sum(kueue_cluster_queue_nominal_quota) by (resource, cluster_queue))
```

Comment:
My only question left is where the 10x comes from? Is this just an arbitrary multiplier which we landed on?

Reply:
This alert was suggested by Antonin and x 10 may have been an arbitrary choice but I think it works. I guess it is possible to line up that many workloads that you could be requesting 10 times too much? wdyt?